### ISSUE TITLE
Support terraform 0.14

### ISSUE THREAD

**Issue by bohui** [terraform-aws-modules/terraform-aws-vpc#544]
Hi,

I installed terraform(v0.14.2) with mac homebrew. Then try to run `terraform init` with https://github.com/k-mitevski/terraform-k8s/blob/master/01_terraform_eks/main.tf

It raised error with

```bash
Error: Unsupported Terraform Core version

  on .terraform/modules/vpc/versions.tf line 2, in terraform:
   2:   required_version = ">= 0.12.7, < 0.14"

Module module.vpc (from terraform-aws-modules/vpc/aws) does not support
Terraform version 0.14.2. To proceed, either choose another supported
Terraform version or update this version constraint. Version constraints are
normally set for good reason, so updating the constraint may lead to other
errors or unexpected behavior.```

**Comment by unbelauscht** [comment]
I am seeing the same thing, even after deleting the `.terraform` directory.

My provider.tf:

```
provider "aws" {
  required_providers = "~> 2.64"
  profile = "terraform-aws-dev"
  region  = "eu-west-1"
}
```

```
tf init         
Initializing modules...
Downloading terraform-aws-modules/vpc/aws 2.48.0 for vpc...
- vpc in .terraform/modules/vpc

Error: Unsupported Terraform Core version

  on .terraform/modules/vpc/versions.tf line 2, in terraform:
   2:   required_version = ">= 0.12.7, < 0.14"

Module module.vpc (from terraform-aws-modules/vpc/aws) does not support
Terraform version 0.14.2. To proceed, either choose another supported
Terraform version or update this version constraint. Version constraints are
normally set for good reason, so updating the constraint may lead to other
errors or unexpected behavior.
```

**Comment by ismailyenigul** [comment]
Hi,
Use version >= 2.58.0.
Terraform 0.14 feature was  added in  v2.58.0.
https://github.com/terraform-aws-modules/terraform-aws-vpc/releases/tag/v2.58.0

**Comment by NasrEddineDev** [comment]
> Hi,
> Use version >= 2.58.0.
> Terraform 0.14 feature was added in v2.58.0.
> https://github.com/terraform-aws-modules/terraform-aws-vpc/releases/tag/v2.58.0

thunks a lot, ربي يبارك فيك

**Comment by bryantbiggs** [comment]
looks like this is resolved when using module version `>= 2.58.0`
